spec/models/user_spec.rb

```diff
@@ -17,5 +17,14 @@
     let(:user) { User.create_by_admin(email: 'lola@o2.pl')}
     it { user.valid_password?('secret').eql? true }
   end
+
+  describe '#to_csv' do
```

Comment:
UsersCsvExport to_csv i use method generat on UsersCsvExport.new with collection of User.includes(:interests)